### DISCUSSION TITLE
Error: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found

### DISCUSSION THREAD

**Discussion by WilliamTambellini** [gitpod-io/openvscode-server#193] [Q&A]
Does this issue occur when all extensions are disabled?: Yes

Error: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found 
required by ~/Downloads/openvscode-server-v1.61.0-linux-x64/node_modules/spdlog/build/Release/spdlog.node

- VS Code Version: 1.61.0
- OS Version: CentOS 7

Steps to Reproduce:
1. follow the guide in the readme
```cpp
tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
./server.sh
```
2. see error

You are apparently considering that the OS libstdc++ is super recent, which not the case on most todays production machines (CentOS7, ubuntu18, ...).
You just have to embed/package the libstdc++ from gcc that you have built with.

**Comment by akosyakov** [comment]
Please see requirements here: https://code.visualstudio.com/docs/remote/linux#_remote-host-container-wsl-linux-prerequisites

**Reply by gbodeen** [reply]
The prerequisite listed on that page is `libstdc++ >= 3.4.18`. However in practice it throws errors if 3.4.20 is not available. So there's a mismatch between intended requirements and actual requirements.

**Reply by WilliamTambellini** [reply]
You are right @gbodeen 
The best long term solution is to embed the libstdcpp.so that has been used at build time.
